src/preamble.js

```diff
@@ -212,11 +212,7 @@ function allocate(slab, allocator) {
 #endif
 
   if (allocator == ALLOC_STACK) {
-#if DECLARE_ASM_MODULE_EXPORTS
     ret = stackAlloc(slab.length);
-#else
-    ret = (typeof stackAlloc !== 'undefined' ? stackAlloc : null)(slab.length);
-#endif
```

Comment:
`stackAlloc` arrives from the wasm module. So in theory it could be undefined if we call `allocate()` during startup. But this replaces that with null which would crash... and we should not try to allocate on the stack during startup anyhow. So this line does seem like it was pointless...